src/app/public/modules/file-attachment/file-item.component.spec.ts

```diff
@@ -0,0 +1,232 @@
+import {
+  TestBed,
+  ComponentFixture,
+  async
+} from '@angular/core/testing';
+
+import {
+  expect
+} from '@blackbaud/skyux-builder/runtime/testing/browser';
```

Comment:
This version of `expect` is deprecated; we're going to start telling people to use:

```
import {
  expect
} from '@skyux-sdk/testing';
```

Reply:
done